src/Workspaces/Core/Portable/Diagnostics/DiagnosticAnalysisResult.cs

```diff
@@ -111,47 +110,7 @@ public static DiagnosticAnalysisResult CreateInitialResult(ProjectId projectId)
             projectId,
             version: VersionStamp.Default,
             documentIds: null,
-            isEmpty: true,
-            fromBuild: false);
-    }
-
-    public static DiagnosticAnalysisResult CreateFromBuild(Project project, ImmutableArray<DiagnosticData> diagnostics, IEnumerable<DocumentId> initialDocuments)
```

Comment:
never called.  this is also the only entrypoint that `fromBuild: true` could pass in (hence why we know it's false everywhere).